### ISSUE TITLE
[Feature Request] Filter out androidx.*, com.google.* and common libraries' packages

### ISSUE THREAD

**Issue by pich4ya** [Cyber-Buddy/APKHunt#4]
I would be great if the APKHunt can have an option to filter out some publicly known libraries' packages.

-  androidx.*
- com.google.*
- com.onesignal.*

Or a configuration way to filter them.
Thank you!

**Comment by noraj** [comment]
Yes, maybe the other around. Rather than a blacklist of common libraries, add an option to specify a whitelist of libraries to audit.

The tool is no reporting vulnerabilities but places with potential danger, it's up to the user to review each point. However, since including tons of third party libraries, 99% of the result are false positive. Since the tool is made for manual review, there should be a way to minimize the results to only include the suitable packages. Else the output is unusable as no-one will review 10 thousands entries just for one APK.

@chawdamrunal How do you think it is possible to implement that?

**Comment by stuxctf** [comment]
@pich4ya I created this tool where you can add some of the libraries; 3 are already excluded, but you can add more. Additionally, there are custom rules in JSON so you can add your own. It's not as advanced as APKhunt, but you could see if it works for you.

https://github.com/stuxctf/slythestx

**Comment by pich4ya** [comment]
@stuxctf great work!
